music21/scale/__init__.py

```diff
@@ -2878,6 +2940,36 @@ def __init__(self, tonic=None):
         # note: this changes the previously assigned AbstractDiatonicScale
         # from the DiatonicScale base class
         self._abstract = AbstractMelodicMinorScale()
+        
+        
+class JazzMinorScale(DiatonicScale):
+    '''
+    The jazz minor scale.
+    
+    A synthetic scale identical to the ascending melodic minor. Useful in
+    jazz, especially over harmonies that employ minor-major seventh chords.
+    >>> sc = scale.JazzMinorScale('e4')
+    >>> [str(p) for p in sc.pitches]
+    ['E4', 'F#4', 'G4', 'A4', 'B4', 'C#5', 'D#5', 'E5']
+    >>> sc.getTonic()
+    <music21.pitch.Pitch E4>
+    >>> sc.getDominant()
+    <music21.pitch.Pitch B4>
+    >>> sc.pitchFromDegree(1)  # scale degree 1 is treated as lowest
```

Comment:
usually no need for comments in doc tests, write it as documentation just above.

Reply:
`<music21.pitch.Pitch B4>`
`>>> sc.pitchFromDegree(1)  # scale degree 1 is treated as lowest`
`<music21.pitch.Pitch E4>`
`>>> sc = scale.JazzMinorScale()`

> explain what's happening here.
usually no need for comments in doc tests, write it as documentation just above.

To clarify, these comments are largely borrowed from other diatonic scales (mainly the harmonic minor) that I had used as a template. Should I rework the comments on these other scales as well?